bigchaindb/config_utils.py

```diff
@@ -91,7 +91,11 @@ def file_config(filename=None):
               file at CONFIG_DEFAULT_PATH, if filename == None)
     """
     logger.debug('On entry into file_config(), filename = {}'.format(filename))
-    if not filename:
+
```

Comment:
Can you explain the code below?
It seems that `filename` should only be a file name or `None`.
Why do we check if it is false?